### PR TITLE
Feature: Ability to override the default docker-compose version.

### PR DESCRIPTION
**Pull request by jaudiger**
Resolve https://github.com/getporter/docker-compose-mixin/issues/5

### CODE REVIEW

**Comment by jaudiger** [comment]
Do not merge as it. Waiting before the PR of @vdice to be merged, and I'll do a rebase to take in account the new way to download the docker-compose tool.

**Review by vdice — COMMENTED** [review]
Thanks @jaudiger!  This is fantastic.  With this feature, we'll have plenty of improvements for the next release.

Some of the pending PRs in this repo may need to wait until next week when the other maintainer Carolyn is back from vacation.  Hopefully we can cut a release soon after.

In the meantime, would you be able to sign your commit(s) for this PR for the DCO check?  See https://github.com/getporter/porter/blob/main/CONTRIBUTING.md#signing-your-commits -- let me know if I can help.

File: README.md
Comment (jaudiger) [reply README.md]: Ooops, fixed ! I also updated:

```yaml
required:
  - docker
      privileged: false
```

to

```yaml
required:
  - docker:
      privileged: false
```

**Comment by jaudiger** [comment]
I updated the PR with your suggestions, thanks!

> Some of the pending PRs in this repo may need to wait until next week when the other maintainer Carolyn is back from vacation. Hopefully, we can cut a release soon after.

That's sounds good to me! Just let me know if I can do anything else. Just one last thing, next week, it'll be my turn to be on vacation :')

File: README.md
Comment (vdice) [reply README.md]: Nice, thank you!

**Comment by vdice** [comment]
> Do not merge as it. Waiting before the PR of @vdice to be merged, and I'll do a rebase to take in account the new way to download the docker-compose tool.

@jaudiger (when back from vacay 🏖️ ), https://github.com/getporter/docker-compose-mixin/pull/18 has been merged, so this PR is ready for rebase when back.  Thanks!

**Comment by jaudiger** [comment]
> > Do not merge as it. Waiting before the PR of @vdice to be merged, and I'll do a rebase to take in account the new way to download the docker-compose tool.
> 
> @jaudiger (when back from vacay  ), #18 has been merged, so this PR is ready for rebase when back. Thanks!

@vdice I'm back from vacation :) I just rebased and forced push on the current PR. Feel free to review it !

**Review by vdice — APPROVED** [review]
LGTM!  Thank you @jaudiger!

**Pull request merged**
